lib/bytes_by_prefix.c

```diff
@@ -62,12 +62,16 @@
 #define B_MULT	BSIZE		/* block size */
 #elif sgi
 #define B_MULT	BBSIZE		/* block size */
-#elif defined(__linux__) || defined(__sun) || defined(__hpux)
+#elif (defined(__linux__) || defined(__sun) || defined(__hpux)) && defined(DEV_BSIZE)
 #define B_MULT	DEV_BSIZE	/* block size */
```

Comment:
Most of the code supporting old UNIXes was removed already, there are just four remaining files in the old test library that were not cleaned up. Feel free to send a patches for these.